src/seed-array.ts

```diff
@@ -1,13 +1,21 @@
-import { JSONSchema7Definition } from 'json-schema'
 import { invariant } from 'outvariant'
 import { faker } from '@faker-js/faker'
 import { seedSchema } from './seed-schema.js'
+import type { JSONSchema } from './types.js'
 
-export function seedArray(schema: JSONSchema7Definition): Array<unknown> {
+export function seedArray(schema: JSONSchema | boolean): Array<unknown> {
```

Comment:
Afaik, `JSONSchema7Definition` is precisely that type union. Why replace?

Reply:
Because we need own type, JSONSchema7 doesn't contain `example` field, but it is allowed by [openapi spec](https://spec.openapis.org/oas/v3.0.3#fixed-fields-19)